modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionServiceTest.java

```diff
@@ -57,10 +68,32 @@ public class ObjectDefinitionServiceTest {
 	@Before
 	public void setUp() throws Exception {
 		_adminUser = TestPropsValues.getUser();
+
+		_objectFolder = _objectFolderLocalService.addObjectFolder(
```

Comment:
Should we delete objcetFolder in `tearDown() `method?

Reply:
In other tests as: `ObjectActionServiceTest`, `ObjectLayoutServiceTest`, `ObjectRelationshipServiceTest`. The variables created on the `setUp()` don't delete on the `tearDown()`

Reply:
I will add `@DeleteAfterTestRun` in the variable.